router/pkg/config/config.schema.json

```diff
@@ -1271,7 +1275,7 @@
         },
         "execution_plan_cache_size": {
           "type": "integer",
-          "default": 10000,
+          "default": 1024,
```

Comment:
Changing default can be unpredictable for customers. How likely is it that this can have an impact?

Reply:
1024 plans cached should be more than enough for most setups. I don't expect any negative impacts.